tests/integration-tests/tests/configure/test_pcluster_configure.py

```diff
@@ -41,7 +41,6 @@ def test_pcluster_configure(
         vpc_stack.cfn_outputs["VpcId"],
         vpc_stack.cfn_outputs["PublicSubnetId"],
         vpc_stack.cfn_outputs["PrivateSubnetId"],
-        vpc_stack,
```

Comment:
Why are we removing this arg?

Reply:
Just because it is unused. So I cleaned this up.